test/Frame.spec.jsx

```diff
@@ -11,7 +11,7 @@ describe('The Frame Component', () => {
 
   afterEach(() => {
     if (div) {
-      div.parentNode.removeChild(div);
+      div.remove();
```

Comment:
Made this simpler since it's covered by all "reasonable" browsers now.  The added test actually would cause this to fail since `parentNode` was null.  So instead of making `div = null` at the end of that test, I just updated this.